Sensors/main.py

```diff
@@ -1,7 +1,26 @@
 #!usr/bin/env python3
 #-*-coding:utf-8-*-
 
+import time
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning) #in order to avoid warning for each http request
+
+from httpRequester import postTemperature
+from TemperatureSimulator import *
+
+#this program is run by the sensors giving temperature, it does not asks for settings
 def run():
     print("hello world!")
+    tS = TemperatureSimulator()
+    while(True):
+        time.sleep(1)

```

Comment:
The sleep time should be parametized by using for example `sys.argv` or anything else. With this change, we could change the sleeping time when running the script (i.e. when booting the sensor)